.travis.yml

```diff
@@ -3,18 +3,7 @@ sudo: false
 bundler_args: ""
 cache: bundler
 rvm:
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
+  - 2.5.8
```

Comment:
What about [2.6](https://github.com/zendesk/zendesk_channels/blob/master/.travis.yml#L6)?

Reply:
We're not testing classic against 2.6 yet, so I only added this one.

Based on the code here, 2.6 and 2.7 will be fine. This is really simple stuff.